### ISSUE TITLE
Error: Summary parsing failed

### ISSUE THREAD

**Issue by Chloe-Woahie** [Michael-F-Bryan/mdbook-epub#58]
When trying to compile [The Rust Async Book](https://github.com/rust-lang/async-book), it gives me a Summary Parsing Failed Error.
```
$ mdbook-epub --standalone async-book
Error: Summary parsing failed
```

I tried running this in both the root directory and the source directory.

**Comment by dieterplex** [comment]
Looks like empty links are not acceptable. One workaround would be removing all items with empty link in `SUMMARY.md`, or maybe fix it with a option whether the links should be handled.

```rust
$ RUST_LOG=trace mdbook-epub -s .
(...skip)
TRACE mdbook_epub::generator] Adding chapter "6. Executing Multiple Futures at a Time"
DEBUG handlebars::render] Rendering value: Path(Relative(([Named("title")], "title")))
DEBUG handlebars::render] Rendering value: Path(Relative(([Named("stylesheet")], "stylesheet")))
DEBUG handlebars::render] Rendering value: Path(Relative(([Named("body")], "body")))
TRACE mdbook_epub::generator] add a chapter "Executing Multiple Futures at a Time" by a path = "06_multiple_futures/01_chapter.md"
TRACE mdbook_epub::generator] add sub-item = "join!"
DEBUG handlebars::render] Rendering value: Path(Relative(([Named("title")], "title")))
DEBUG handlebars::render] Rendering value: Path(Relative(([Named("stylesheet")], "stylesheet")))
DEBUG handlebars::render] Rendering value: Path(Relative(([Named("body")], "body")))
TRACE mdbook_epub::generator] add a chapter "join!" by a path = "06_multiple_futures/02_join.md"
TRACE mdbook_epub::generator] add sub-item = "select!"
DEBUG handlebars::render] Rendering value: Path(Relative(([Named("title")], "title")))
DEBUG handlebars::render] Rendering value: Path(Relative(([Named("stylesheet")], "stylesheet")))
DEBUG handlebars::render] Rendering value: Path(Relative(([Named("body")], "body")))
TRACE mdbook_epub::generator] add a chapter "select!" by a path = "06_multiple_futures/03_select.md"
TRACE mdbook_epub::generator] add sub-item = "TODO: Spawning"
ERROR mdbook_epub] No CSS found by a path =  = None
```

part of SUMMARY.md
```markdown
- [Executing Multiple Futures at a Time](06_multiple_futures/01_chapter.md)
  - [`join!`](06_multiple_futures/02_join.md)
  - [`select!`](06_multiple_futures/03_select.md)
  - [TODO: Spawning]()
  - [TODO: Cancellation and Timeouts]()
  - [TODO: `FuturesUnordered`]()
```

https://github.com/Michael-F-Bryan/mdbook-epub/blob/23b4f766700d08d404bb6d937f2c050381b76a06/src/generator.rs#L145-L146

**Comment by blandger** [comment]
Fixed and tested on that book in [PR #73](https://github.com/Michael-F-Bryan/mdbook-epub/pull/73)
